src/lib/utils.js

```diff
@@ -1,21 +1,29 @@
 import { SHARE_SITES } from './enums';
 
-
 /**
  * Converts Date String with UTC timezone to date consumable by calendar
  * apps. Changes +00:00 to Z.
  * @param {string} Date in YYYYMMDDTHHmmssZ format
  * @returns {string} Date with +00:00 replaceed with Z
  */
-export const formatDate = date => date && date.replace('+00:00', 'Z');
+export const formatDate = (date) => date && date.replace('+00:00', 'Z');
 
+/**
+ * Lets you know if msSaveOrOpenBlob should be used to download the ical file
+ */
+export const isInternetExplorer = () => {
+  return window.navigator && window.navigator.msSaveOrOpenBlob && window.Blob;
```

Comment:
I think we should test if the browser is IE against the userAgent, like how we're testing mobile. Something like this:

```
export const isInternetExplorer = /MSIE/.test(window.navigator.userAgent) || /Trident/.test(window.navigator.userAgent);
```

Resources: http://www.useragentstring.com/pages/useragentstring.php?name=Internet+Explorer